src/tools/recvfuzz/recvfuzz.cpp

```diff
@@ -1107,7 +1107,7 @@ void SetupAndFuzz() {
 
     CxPlatSocketDelete(Binding);
     CxPlatDataPathUninitialize(Datapath);
-    CxPlatWorkerPoolDelete(WorkerPool);
+    CxPlatWorkerPoolDelete(WorkerPool, CXPLAT_WORKER_POOL_REF_LIBRARY);
```

Comment:
These refs in tools make me think that we should:
- Add a ref type parameter to `CxPlatWorkerPoolCreate`
- Create a ref type for tools

Just to be consistent